ksql-engine/src/test/java/io/confluent/ksql/query/QueryExecutorTest.java

```diff
@@ -310,20 +311,20 @@ public void shouldMaterializeCorrectly() {
     // When:
     final PersistentQueryMetadata queryMetadata = queryBuilder.buildQuery(
         STATEMENT_TEXT,
-        QUERY_ID,
+        PULL_QUERY_ID,
```

Comment:
This should be `QUERY_ID` as this is the id of the query that is building the materialized state, not the id of the pull query accessing it.

Reply:
I believe that this should be actually the pull query ID because we do `queryMetadata.getMaterialization()` which is only called by pull queries.